packages/zoltan2/test/connectivity/meshData/three-borders.quad.msh

```diff
@@ -0,0 +1,5 @@
+nonsense line
+0 0 3
```

Comment:
This file does not follow the file format, which prescribes that line 3 must be blank.  Does it actually run correctly?  If so, why?

Reply:
The format has changed, so this is no longer an issue.